demo-shell/src/app/components/app-layout/cloud/people-groups-cloud-demo.component.html

```diff
@@ -16,13 +16,17 @@
                 <mat-label>{{ 'PEOPLE_GROUPS_CLOUD.ROLE' | translate }} ['ACTIVITI_ADMIN', "ACTIVITI_USER"]</mat-label>
                 <input matInput (input)="setPeopleRoles($event)" />
             </mat-form-field>
-            <mat-form-field class="adf-preselect-value full">
+            <mat-form-field class="adf-preselect-value-big">
+                    <mat-label>{{ 'PEOPLE_GROUPS_CLOUD.APP_NAME' | translate }}</mat-label>
+                    <input matInput (input)="setPeopleAppName($event)" />
+                </mat-form-field>
+            <mat-form-field class="adf-preselect-value-full">
                 <mat-label>{{ 'PEOPLE_GROUPS_CLOUD.PRESELECTED_VALUE' | translate }}: {{ DEFAULT_PEOPLE_PLACEHOLDER }}</mat-label>
                 <input matInput (input)="setPeoplePreselectValue($event)" />
             </mat-form-field>
         </div>
         <div>
-            <adf-cloud-people [preSelectUsers]="preSelectUsers" [roles]="peopleRoles" [mode]="peopleMode"></adf-cloud-people>
+            <adf-cloud-people [preSelectUsers]="preSelectUsers" [appName]="peopleAppName" [roles]="peopleRoles" [mode]="peopleMode"></adf-cloud-people>
```

Comment:
Can you please add a radio button that says appNameVisible 
if is true we can show the input filed appName and hide the roles.